torchgeo/datamodules/vhr10.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""NWPU VHR-10 datamodule."""
+
+from typing import Any, Callable, Union
+
+import kornia.augmentation as K
+import torch
+from einops import rearrange
+from torch import Generator, Tensor
+from torch.nn import Module
+from torch.utils.data import random_split
```

Comment:
Let's just use `.utils.dataset_split` instead so we don't need to bump the minimum version of torch/torchvision.

Reply:
Documentation of that function says it was deprecated in 0.4 but I imagine the aim is to replace it later everywhere in one go.